CHANGELOG.md

```diff
@@ -176,6 +176,13 @@ is based on [Keep a Changelog](https://keepachangelog.com).
   assume that the parsed URI for `tcp://127.0.0.1:8080` returns an IP address
   when calling `authority().host`.
 
+## [0.17.7] - Unreleased
```

Comment:
Missing link. Please add

```md
[0.17.7]: https://github.com/actor-framework/actor-framework/compare/0.17.6...release/0.17
```

at the bottom of the `.md` file at the appropriate position.